test/gem-empty/command_test.rb

```diff
@@ -18,11 +19,11 @@ def self.remove_spec spec
   end
 
   it "has some strings" do
-    subject.arguments.class.must_equal(String)
-    subject.usage.class.must_equal(String)
-    subject.defaults_str.class.must_equal(String)
-    subject.description.class.must_equal(String)
-    subject.program_name.class.must_equal(String)
+    _(subject.arguments.class).must_equal(String)
```

Comment:
What's these `_` calls for?

Reply:
As written in the commit message, this was the minitest recommendation:
```
DEPRECATED: global use of wont_equal from test/gem-empty/specification_and_version_test.rb:24. Use _(obj).wont_equal instead. This will fail in Minitest 6.
```